lib/matplotlib/scale.py

```diff
@@ -53,6 +53,7 @@ def __init__(self, axis):
         be used: a single scale object should be usable by multiple
         `~matplotlib.axis.Axis`\es at the same time.
         """
+        self._kwargs = dict()
```

Comment:
Norms now make themselves out of a scale.  When the scale is created, some parameters are passed, and we need those if we are to re-recreate the scale for a colorbar.